annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -501,13 +507,21 @@ protected void validateSubtitleOrCaption(Annotation webAnnotation)
 	// check mandatory field body.format (please note that this field is saved in the "contentType" field of the given Resource)
 	if (StringUtils.isBlank(body.getContentType())) {
 	    throw new PropertyValidationException(I18nConstants.MESSAGE_MISSING_MANDATORY_FIELD,
-		    I18nConstants.MESSAGE_MISSING_MANDATORY_FIELD, new String[] { "subtitle.body.format" });
+		    I18nConstants.MESSAGE_MISSING_MANDATORY_FIELD, new String[] { "body.format" });
 	}
 	// check if the body.format field has a valid value
 	boolean result = getConfiguration().getAnnotationSubtitlesFormats().contains(body.getContentType());
 	if (!result) {
 	    throw new PropertyValidationException(I18nConstants.ANNOTATION_INVALID_SUBTITLES_FORMATS,
-		    I18nConstants.ANNOTATION_INVALID_SUBTITLES_FORMATS, new String[] { body.getContentType() });
+		    I18nConstants.ANNOTATION_INVALID_SUBTITLES_FORMATS, new String[] { "body.format" });
+	}
+	// check if the body.value is valid 
+	try {
```

Comment:
please move this validation to own method, validateSubtitleContent and do not enable the execution until the spike ticket is concluded

Reply:
Done.